03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,18 @@
+Q #1
+A request handler with access to the application's 
+request-response cycle is known as middleware. It's a 
+function that holds the request object, the response object, 
+and the middleware function. Middleware can also send the response 
+to the server before the request.
+
+Q #2
+CREATE, READ, UPDATE, and DELETE
+CRUD is the acronym for CREATE, READ, UPDATE and DELETE. These
+terms describe the four essential operations for creating and 
+managing persistent data elements, mainly in relational and 
+NoSQL databases.
```

Comment:
https://www.crowdstrike.com/cybersecurity-101/observability/crud/

Can you rewrite this in your own words?